google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerImpl.java

```diff
@@ -146,16 +147,23 @@
   @GuardedBy("this")
   private boolean spannerIsClosed = false;
 
-  SpannerImpl(SpannerRpc rpc, int defaultPrefetchChunks, SpannerOptions options) {
+  SpannerImpl(
+      SpannerRpc rpc, SpannerRpc gapicRpc, int defaultPrefetchChunks, SpannerOptions options) {
     super(options);
     this.rpc = rpc;
+    this.gapicRpc = gapicRpc;
```

Comment:
Could you rename the existing `rpc` to something like `rawGrpcRpc` to make the contrast clearer?

Reply:
Done